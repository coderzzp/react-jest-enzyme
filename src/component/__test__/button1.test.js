// snapshot
import React from 'react'
import { shallow } from 'enzyme';

describe('Button 组件测试', () => {
  it('渲染正常', () => {
    const Button = require('../button').default
    const componentWrapper = shallow(<Button />);
    expect(componentWrapper.html()).toMatchSnapshot();
  });
  it('props传入buttonName,渲染正常', () => {
    const Button = require('../button').default
    const componentWrapper = shallow(<Button buttonName={'mockButtonName'}/>);
    expect(componentWrapper.html()).toMatchSnapshot();
  });
});