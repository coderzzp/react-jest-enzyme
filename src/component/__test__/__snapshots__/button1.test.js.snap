// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 组件测试 props传入buttonName,渲染正常 1`] = `"<div><div class=\\"button\\">mockButtonName</div></div>"`;

exports[`Button 组件测试 渲染正常 1`] = `"<div><div class=\\"button\\">buttonDefaultName</div></div>"`;
